daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -138,7 +138,10 @@ General guidance on upgrading can be found for [self hosted mode]({{< ref self-h
 | 1.10.0                   |                   N/A |                   1.10.8 |
 | 1.11.0                   |                   N/A |                   1.11.4 |
 | 1.12.0                   |                   N/A |                   1.12.4 |
-| 1.13.0                   |                   N/A |                   1.13.5 |
+| 1.13.0                   |                   N/A |                   1.13.2 |
```

Comment:
This need updating. You only need these entries

1.13.0 can go to 1.13.4
"1.13.0 to 1.13.4"  can go to 1.14.0

Also we seem to be missing the other ranges. At least add this one
 "1.12.0 to 1.12.4" can go to 1.13.4